mesonbuild/coredata.py

```diff
@@ -460,12 +460,17 @@ def set_options(self, options, subproject=''):
             sub = 'In subproject {}: '.format(subproject) if subproject else ''
             mlog.warning('{}Unknown options: "{}"'.format(sub, unknown_options))
 
+class CmdLineFileParser(configparser.ConfigParser):
```

Comment:
It might be nice to have a comment in the class explaining why we only want = to be a delimiter. Other than that this seems like a good change.

Reply:
done